iCubGenova09/hardware/mechanicals/right_leg-eb12-j4_5-mec.xml

```diff
@@ -11,7 +11,7 @@
         <param name="Encoder">          182.044              182.044       </param>
         <param name="fullscalePWM">    32000               32000           </param>
         <param name="ampsToSensor">     1000.0              1000.0         </param>
-        <param name="Gearbox_M2J">       100.0               100.0         </param>
+        <param name="Gearbox_M2J">       100.0               160.0         </param>
```

Comment:
Already in `devel`?!
See https://github.com/robotology/robots-configuration/blob/devel/iCubGenova09/hardware/mechanicals/right_leg-eb12-j4_5-mec.xml#L14